plugins/mlkit-digital-ink-recognition.md

```diff
@@ -16,8 +16,8 @@ npm install @nativescript/mlkit-digital-ink-recognition
 
 ## Usage
 
-See [@nativescript/mlkit-core](/packages/mlkit-core/README.md) Usage
-
+See [@nativescript/mlkit-core](/plugins/mlkit-core.html#core) Usage
+plugins/mlkit-core.html#core
```

Comment:
These changes will be overwritten from the plugins README, so would need to be made in the plugins repo instead.

Reply:
I am aware of that. You informed me about it when I made my first PR for a plugin. Thanks. Initially, I made these changes in the plugins' repo. However, when their READMEs were added to the main docs, the links broke(they only work on Github) because when you click on them from `https://docs.nativescript.org/` they become `https://docs.nativescript.org/packages/mlkit-core/README.html` and since the  `packages` folder is not in  `docs-new` directory, they break.  How can I make them work on both Github(when navigating from one README to another, within the same package) and `https://docs.nativescript.org/`?